docs/12.0/guides/upgrade-and-migration/release-notes.md

```diff
@@ -49,7 +49,7 @@ Released on 28th of April, 2022
 - Changed two scripts of the main Handsontable workspace (`./`), to speed up the build process: now, the `npm run build` and `npm run test` scripts don't build or test the Handsontable examples (`./examples`). [#9412](https://github.com/handsontable/handsontable/issues/9412)
 - Changed the version of the Moment.js dependency from 2.24.0 to 2.29.3, in reaction to a recently-found Moment.js security vulnerability that did not directly affect Handsontable. [#9381](https://github.com/handsontable/handsontable/issues/9381)
 - Changed the version of the HyperFormula optional dependency from ^1.2.0 to ^2.0.0. [#9411](https://github.com/handsontable/handsontable/issues/9411)
-- Changed an internal property name, from `hot.view.wt` to `hot.view._wt`, where `hot` is the name of your Handsontable instance. The intention of this change is to make it clear that Walkontable (Handsontable's rendering engine) is not a part of Handsontable's public API. [#8760](https://github.com/handsontable/handsontable/issues/8760)
+- Changed an internal property name, from `handsontableInstance.wt` to `handsontableInstance._wt`, to make it clear that Walkontable (Handsontable's rendering engine) is not a part of Handsontable's public API. [#8760](https://github.com/handsontable/handsontable/issues/8760)
```

Comment:
Almost there! But the `view.` part is missing. It should be `handsontableInstance.view.wt` / `handsontableInstance.view._wt` everywhere.

Reply:
@warpech - 🤦 done: [ea4f9f7](https://github.com/handsontable/handsontable/pull/9550/commits/ea4f9f7dbc08b6721fcf28981e83e06d70bdd007)